aspnetcore/blazor/components.md

```diff
@@ -1747,6 +1862,32 @@ A limited set of ASP.NET Core's localization scenarios are currently supported:
 
 For more information, see <xref:fundamentals/localization>.
 
+### Configure the linker for internationalization (Blazor WebAssembly)
```

Comment:
This section should go in the [Configure the LInker](https://docs.microsoft.com/aspnet/core/host-and-deploy/blazor/configure-linker) topic.